File: java/Napakalaki/src/NapakalakiGame/Cultist.java
package NapakalakiGame;

/**
 * 
 * @author Sofía Almeida Bruno
 * @author María Victoria Granados Pozo
 */
public class Cultist {
    private String name;
    private int gainedLevels;
    
    /**
     * 
     * Constructor
     * @param name Nombre del sectario
     * @param gainedLevels Niveles añadidos 
     */
    public Cultist(String name, int gainedLevels) {
        this.name = name;
        this.gainedLevels = gainedLevels;
    }
    
    /**
     * 
     * @return Niveles que se añaden por ser sectario
     */
    public int getGainedLevels() {
        return gainedLevels;
    }
}

File: ruby/Napakalaki/lib/cultist.rb
#encoding: utf-8

#Sofía Almeida Bruno
#María Victoria Granados Pozo

module NapakalakiGame
  class Cultist
    def initialize(name, gainedLevels)
      @name = name
      @gainedLevels = gainedLevels
    end
    
    def getGainedLevels
      @gainedLevels
    end  
  end
end
